amida-tech/amida-auth-microservice
# [SER-254] Seed Admin Username Joi Support Email

#### What's this PR do?

Previously, Joi validation required that `AUTH_SERVICE_SEED_ADMIN_EMAIL` be an email address but `AUTH_SERVICE_SEED_ADMIN_USERNAME` be alphanumeric.

We want to be able to set both to the same value. Therefore, Joi validation for `AUTH_SERVICE_SEED_ADMIN_USERNAME` was changed to an email address.

#### Related JIRA tickets:

https://jira.amida.com/browse/SER-254

#### How should this be manually tested?

Set `AUTH_SERVICE_SEED_ADMIN_USERNAME` to something that isn't an email address and isn't a plain alphanumeric, such as `a.b.c`. Joi validation should fail.

Set it to an email address or to an alphanumeric value. Joi validation should pass.

#### Any background context you want to provide?

On all deployments of Orange, it was continually frustrating to manage the `_USERNAME` and `_EMAIL` being different values.

- [x] Passes all tests
